recipes/oracle-jdk/post-link.bat

```diff
@@ -0,0 +1,46 @@
+
+@echo off
+
+echo "Installing in %CONDA_PREFIX%" > "%CONDA_PREFIX%\.messages.txt"
+echo "   CONDA_PREFIX: %CONDA_PREFIX%" >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_NAME:     %PKG_NAME%"     >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_VERSION:  %PKG_VERSION%"  >> "%CONDA_PREFIX%\.messages.txt"
+echo "   PKG_BUILDNUM: %PKG_BUILDNUM%" >> "%CONDA_PREFIX%\.messages.txt"
+:: set | FindStr "PKG" >> "%CONDA_PREFIX%\.messages.txt"
+
+set "PKG_BIN=%CONDA_PREFIX%\bin"
+set "PKG_UUID=%PKG_NAME%-%PKG_VERSION%_%PKG_BUILDNUM%"
+
+:: should I be using `reg query ` to find the jdk directory path?
+set "ORACLE_JDK_DIR=%ProgramFiles%\Java\jdk1.8.0_321"
+
+:: Discovery
```

Comment:
Given how often the revision for v8 changes. It would be better to find the latest revision. 
i.e. match on jdk1.8.0* and not the specific jdk1.8.0_321

Ideally, I would like to discover the ORACLE_JDK_DIR in the `post-link` and then make it available to `pre-unlink`, `activate`, and `deactivate`. Is there a way to do that?

Reply:
IDK if there is an official way to do it. I am sure putting a file with the path in the prefix would work fine.

Reply:
I used a similar approach. `post-link` writes a file to the "conda-meso" folder.
```
set "CONDA_MESO=%CONDA_PREFIX%\conda-meso\%PKG_NAME%-%PKG_VERSION%_%PKG_BUILDNUM%"
set "DISCOVERY_SCRIPT=%CONDA_MESO%\discovery.bat"
```

Reply:
One thing here is that the `conda-*` namespace in the PREFIX should not be used by packages. We should pick something very obvious that reflects the package that is dealing with the files, e.g. `oracle-jdk-meta` or whatever. This makes it painfully obvious where files might be if we are debugging user environments later.

Reply:
I am trying to parallel the pattern set by `conda-meta` i.e. `%CONDA_PREFIX%\conda-meta\%PKG_NAME%-%PKG_VERSION%_%PKG_BUILDNUM%.json"
I have several packages that I need to do a similar treatment on and I do not think cluttering the `%CONDA_PREFIX%` would be wise.

Reply:
Hmmm. The vast majority of environments won't have these packages installed and so this didn't concern me. 

Maybe @conda-forge/core has thoughts?